sensors/omron_d6t/omron_d6t.py

```diff
@@ -38,15 +44,15 @@ def __read(self):
         """
 
         # This loop will run until a valid value is received from the sensor (or max retries is reached)
-        for i in range(0, self.MAX_RETRIES):
+        for i in range(0, self._MAX_RETRIES):
             bytes_read, sensor_raw_data = self.read_raw_data()
 
             if bytes_read == self.__BUFFER_LENGTH:
                 # Read PEC byte for checksum
                 crc_expected = sensor_raw_data[34]
```

Comment:
Magic number

Reply:
I made changes to use -1, but is -1 still considered a magic number?